packages/lib-classifier/src/store/UserProjectPreferencesStore/UserProjectPreferencesStore.js

```diff
@@ -56,11 +56,10 @@ const UserProjectPreferencesStore = types
       const { authClient } = getRoot(self)
 
       try {
-        const authorization = yield getBearerToken(authClient)
```

Comment:
I'm not sure if this can be reliably reproduced, but I was seeing this occasionally return an empty string, even when I was logged in.